app/constants/Strings.ts

```diff
@@ -42,3 +42,18 @@ export default {
     profile: 'Profile',
   },
 };
+export enum NavigationRoutes {
+  SplashScreen = 'SPLASHSCREEN',
```

Comment:
use underscore for example SPLASH_SCREEN , apply on others

Reply:
resolved